examples/src/main/java/com/mapbox/navigation/examples/settings/OfflineRegionDownloadActivity.kt

```diff
@@ -32,16 +31,11 @@ import com.mapbox.mapboxsdk.style.sources.GeoJsonSource
 import com.mapbox.navigation.base.logger.model.Message
 import com.mapbox.navigation.examples.R
 import com.mapbox.navigation.logger.MapboxLogger
-import com.mapbox.services.android.navigation.v5.navigation.MapboxOfflineRouter
-import com.mapbox.services.android.navigation.v5.navigation.OfflineError
-import com.mapbox.services.android.navigation.v5.navigation.OfflineTiles
-import com.mapbox.services.android.navigation.v5.navigation.OnOfflineTilesRemovedCallback
-import com.mapbox.services.android.navigation.v5.navigation.OnTileVersionsFoundCallback
-import com.mapbox.services.android.navigation.v5.navigation.RouteTileDownloadListener
+import com.mapbox.navigation.route.onboard.OnOfflineTilesRemovedCallback
 import kotlinx.android.synthetic.main.activity_offline_region_download.*
 import org.json.JSONObject
 
-class OfflineRegionDownloadActivity : AppCompatActivity(), RouteTileDownloadListener,
+class OfflineRegionDownloadActivity : AppCompatActivity(), /*RouteTileDownloadListener,*/
```

Comment:
don't have opportunity to download tiles now. https://github.com/mapbox/mapbox-navigation-android/issues/2493 . Suggest to remove this activity

Reply:
Yeah will remove this activity. I will also be removing `MockNavigationActivity`